### PR TITLE
[SCB-260] Add third party notice

### PR DESCRIPTION
**Pull request by yangbor**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [x] Run `mvn clean install` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

File: java-chassis-distribution/src/release/NOTICE [hunk 1]
```diff
@@ -3,3 +3,48 @@ Copyright 2017-2018 The Apache Software Foundation
...
+
+
+================================================================
+Notice for Apache Commons Codec
+
+See licenses/NOTICE-apache-commons-codec
+
+================================================================
+Notice for Apache Commons Lang
+
```
Comment (WillemJiang) [inline java-chassis-distribution/src/release/NOTICE]: I think there is Copyright line here.

File: java-chassis-distribution/src/release/NOTICE [hunk 2]
```diff
@@ -3,3 +3,48 @@ Copyright 2017-2018 The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+
+================================================================
+Notice for Apache Commons Codec
+
+See licenses/NOTICE-apache-commons-codec
```
Comment (WillemJiang) [inline java-chassis-distribution/src/release/NOTICE]: Please put the the notices directory here.

Comment (yangbor) [reply java-chassis-distribution/src/release/NOTICE]: As explained in other discussion, there is no need to put up a notices directory which will increase end user burden on collecting legal information.

**Comment by WillemJiang** [comment]
We need to remove the duplicated copyright of ASF products. 
I will commit a quick fix for it shortly.

**Pull request merged**